types/relay-runtime/lib/multi-actor-environment/MultiActorEnvironmentTypes.d.ts

```diff
@@ -82,27 +79,18 @@ export interface MultiActorEnvironment {
      * retained in-memory. The records will not be eligible for garbage collection
      * until the returned reference is disposed.
      */
-    retain(
-        actorEnvironment: ActorEnvironment,
-        operation: OperationDescriptor,
-    ): Disposable;
+    retain(actorEnvironment: ActorEnvironment, operation: OperationDescriptor): Disposable;
 
     /**
      * Apply an optimistic update to the environment. The mutation can be reverted
      * by calling `dispose()` on the returned value.
      */
-    applyUpdate(
-        actorEnvironment: ActorEnvironment,
-        optimisticUpdate: OptimisticUpdateFunction,
-    ): Disposable;
+    applyUpdate(actorEnvironment: ActorEnvironment, optimisticUpdate: OptimisticUpdateFunction): Disposable;
```

Comment:
can you please revert these stylistic changes — prefer to keep changes to only what needs to be changed 🙏🏻

Reply:
You gave the [same feedback on a previous PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58345#discussion_r798157234), these files were all formatted by prettier as they should ahve been from the beginning.

I can revert these changes and then apply the formatting changes in a spearate PR, but they _should_ go in at some point so this stops being an issue.

Reply:
ah if there is tooling about this moment (don't keep tabs on what the dts crew change), lets yeah as you say ship formatting change first!